expansion/src/main/java/no/unit/nva/expansion/model/ExpandedDoiRequest.java

```diff
@@ -157,9 +150,25 @@ private static ExpandedDoiRequest fromDoiRequest(DoiRequest doiRequest, Resource
         entry.setCustomerId(doiRequest.getCustomerId());
         entry.setModifiedDate(doiRequest.getModifiedDate());
         entry.setStatus(ExpandedTicketStatusMapper.getExpandedTicketStatus(doiRequest));
-        entry.setViewedBy(doiRequest.getViewedBy());
+        entry.setViewedBy(expandPersonViewedBy(doiRequest.getViewedBy(), resourceExpansionService));
         entry.setPublication(publicationSummary);
-        entry.setFinalizedBy(doiRequest.getFinalizedBy());
+        entry.setFinalizedBy(expandPerson(doiRequest.getFinalizedBy(), resourceExpansionService));
         return entry;
     }
+
+    private static ExpandedPerson expandPerson(Username username,
```

Comment:
Exact same copy of expandPerson and expandPersonViewedBy in three classes. Extract methods in a ExpansionUtil class?